packages/lib-taquito/addHexPrefix.ts

```diff
@@ -0,0 +1,3 @@
+export function addHexPrefix(inputString: string): string {
+    return inputString.startsWith('0x') ? inputString : '0x' + inputString;
+}
```

Comment:
small pet peeve, can we make sure all files end with a new line so we don't get this? :) `addHexPrefix`, `constants` and `uint8ArraysEqual.ts`.

<img width="116" alt="Screenshot 2023-12-11 at 12 54 53 PM" src="https://github.com/dfns/dfns-sdk-ts/assets/360406/db4cf03d-524d-4e66-970c-3010c7868170">